clap_complete/tests/testsuite/engine.rs

```diff
@@ -764,7 +775,15 @@ pos_c
 
     assert_data_eq!(
         complete!(cmd, "--format json -- pos_1 pos_a pos_b pos_c [TAB]"),
-        snapbox::str![]
+        snapbox::str![[r#"
+pos_d
+pos_e
+"#]]
+    );
```

Comment:
The old test specifically had a case to verify that there was nothing more to complete.  We lose that by extending this test.  Instead, can you add a separate test for unbounded positionals?

Reply:
(in doing this, we'd make `positional-2` unbounded)